### ISSUE TITLE
[APP SUBMITTED]: ('Connection aborted.', error(10054, 'An existing connection was forcibly closed by the remote host'))

### ISSUE THREAD

**Issue by titomontes** [SickRage/sickrage-issues#558]
### INFO
Python Version: **2.7.8 (default, Jun 30 2014, 16:03:49) [MSC v.1500 32 bit (Intel)]**
Operating System: **Windows-8-6.2.9200**
Locale: cp1252
Branch: **master**
Commit: SickRage/SickRage@66fa37c556784fa3b41a06bb3043bd854d165c10
Link to Log: https://gist.github.com/2ab09cbf600be1e7e812
### ERROR
```
SEARCHQUEUE-MANUAL-279121 :: [Torrentz] :: qbittorrent: Unable to connect ('Connection aborted.', error(10054, 'An existing connection was forcibly closed by the remote host')) [66fa37c]
```
---
_STAFF NOTIFIED_: @SickRage/owners @SickRage/moderators

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist)

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by fernandog** [comment]
@titomontes check you qbittorrent host/port/user/pass

**Comment by titomontes** [comment]
There is some kind of issue there.

I changed the acces from [localhost] to [IP] and it worked. It appears SR is not accepting [localhost] instead of [IP]

**Comment by fernandog** [comment]
using vpn or proxy or something?

**Comment by titomontes** [comment]
Hello, sorry for the delay

No proxy nor VPN. I tried it out again and it failed to connect again. The first few times it worked (with user and pw entered), deleted user and pw and worked a few times more and then refused connection.
I have tried several times changing back to localhost and again, unchecked bypass authetication on QbT, restarted SR (updates are set every 12 h), went bak to my latest working settings and now it refuses to connect!!
Obviously, qbittorrent and SR are on the same PC, so localhost should work.

I think I´m not doing anything wrong, so there must be a bug somewhere.
